openblas 0.3.13 fails to compile on macOS

openblas 0.3.13 fails to compile in Homebrew (https://github.com/Homebrew/homebrew-core/pull/66821) with:

```
gcc-ranlib ../../libopenblasp-r0.3.13.a
 make[2]: gcc-ranlib: No such file or directory
 make[2]: *** [../../libopenblasp-r0.3.13.a] Error 1
 make[1]: *** [lapacklib] Error 2
 make: *** [netlib] Error 2
```

We run:

```
system make CC=gcc-10 FC=gfortran libs netlib shared
```

I have no idea why it's trying to call `gcc-ranlib`, but we don't have it installed. The system `ranlib` works perfectly fine, and should be called. We can (and probably will) override this with `RANLIB=ranlib` but that still seems like a bug that should be fixed here.

--- a/c_check
+++ b/c_check
@@ -5,7 +5,7 @@
 
 # Checking cross compile
 $hostos   = `uname -s | sed -e s/\-.*//`;    chop($hostos);
-$hostarch = `uname -m | sed -e s/i.86/x86/`;chop($hostarch);
+$hostarch = `uname -m | sed -e s/i.86/x86/`;
 $hostarch = `uname -p` if ($hostos eq "AIX" || $hostos eq "SunOS");
 chop($hostarch);
 $hostarch = "x86_64" if ($hostarch eq "amd64");

